spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -906,6 +909,10 @@ function cancelChannelUpgrade(
     // If counterparty sequence is less than the current sequence,
     // abort transaction since this error receipt is from a previous upgrade
     abortTransactionUnless(errorReceipt.sequence >= channel.upgradeSequence)
+    // fastforward channel sequence to higher sequence so that we can start
+    // new handshake on a fresh sequence
+    channel.upgradeSequence = errorReceipt.sequence
```

Comment:
That's sweet, indeed without this change we are unable to start a fresh upgrade handshake after a `cancelChannelUpgrade` datagram is sent. 
This [PR](https://github.com/cosmos/ibc/pull/1048) propose a different change to achieve the same goal that is in alignment with the implementation.